### DISCUSSION TITLE
Permit a fixed field to be declared inside a readonly struct.

### DISCUSSION THREAD

**Discussion by gafter** [dotnet/csharplang#9020] [Language Ideas]
From #1502 

Permit a fixed field to be declared inside a readonly struct.

**Comment by sweemer** [comment]
For me the primary motivation of allowing a fixed field inside of a readonly struct would be to make it more "ref readonly friendly" as described [here](https://devblogs.microsoft.com/premier-developer/avoiding-struct-and-readonly-reference-performance-pitfalls-with-errorprone-net/#ref-readonly-friendliness).

Not sure if this is the right place to ask but is this the only way to make such structs ref readonly friendly, or might there be a better way?
